db/background_mgr_resync_dcp.go

```diff
@@ -32,6 +33,9 @@ type ResyncManagerDCP struct {
 	lock          sync.RWMutex
 }
 
+// resyncPostReqBody contains map of scope names with collection names against which resync needs to run
+type resyncPostReqBody map[string][]string
```

Comment:
It would be preferable to define the resync post body with a top level 'scopes' property containing this map, to support future extensibility of the payload.

Reply:
In fact, it makes sense to support setting regenerate_sequences=true inside the payload (in addition to being able to set as a query param).

Reply:
Which one will take higher precedence, the query param or one in the request body? or we regenerate sequences if either one is true?